### ISSUE TITLE
Ability to adjust the InputLabelProps on MultiSelectElement

### ISSUE THREAD

**Issue by waltir** [dohomi/react-hook-form-mui#257]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

I'm having difficulties adjusting the InputLabelProps on the MultiSelectElement. Specifically applying the "shrink: true". This was somewhat easy to do to other fields (after looking at the src code), however, with this field i've been unsuccessful even after reviewing the src code. I've attached an example of my MultiSelectElement containing just a small amount of the  things i've tried. If you could point me in the right direction or tell me if this is not currently supported that would be great.

Thanks!

```jsx    
<MultiSelectElement
      name={props.name}
      label={props.label}
      options={props.options}
      size={"small"}
      margin={"dense"}
      fullWidth={true}
      showChips={true}
      variant={"standard"}
      disableUnderline={true}  
      inputProps={{
        shrink: true
        textFieldProps: {
          margin: "dense",
          size: "small",
          variant: "standard",
          InputProps: {
            disableUnderline: true
          },
          InputLabelProps: {
            shrink: true
          }
        },
          InputProps: {
            disableUnderline: true
          },
          InputLabelProps: {
            shrink: true
          }
      }}
      formControlProps={{
        sx: {
          "& .MuiInputBase-root": {
            border: "solid 1px",
            borderRadius: "3px"
          },
          "& .MuiInput-root .MuiInput-input": {
            paddingLeft: "10px",
            marginBottom: "-2px"
          }
        }
      }}
    />
```

![image](https://github.com/dohomi/react-hook-form-mui/assets/38726648/5b5030a9-b6d8-4154-8f33-03e5e6f3a92c)

**Comment by waltir** [comment]
Additionally, I'm having difficulties adjusting the following menuMaxHeight, menuMaxWidth, minWidth. It appears that the values remain the same despite the values I'm passing to the component. If possible can you please demonstrate how these default values / behavior can be overwritten.

```
    menuMaxHeight = ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
    menuMaxWidth = 250,
    minWidth = 120,
```

**Comment by waltir** [comment]
NVM, we ended up deciding to not use this plugin due to the lack of documentation. After previously spending a couple days struggling to customize the fields above the way we needed I simply started with react-hook-form from scratch and had what I needed in an afternoon with clear documentation. 

If anyone else finds this after having similar difficulties simply implement react-hook-form on your own by wrapping your MUI fields in a controller component which is documented here ([IntegratingwithUIlibraries](https://react-hook-form.com/get-started#IntegratingwithUIlibraries)). It's very simple and straight forward and will save you a lot of time in the long run.
